public/js/navbar.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
added a navbar.js (this is changed from the homepage.js I had earlier, I think that was present in the last merge)